samples/php8/ecom-server/public/terminal_payment_form.php

```diff
@@ -84,22 +91,65 @@ function generateRandomOrderId($length = 8) {
         <input type="checkbox" id="show_tips" name="show_tips">
     </div>
 
+    <div style="display: flex; align-items: flex-end; margin-bottom: 20px;">
+        <label for="enable_payment_method">Enable payment method: </label>
+        <input type="checkbox" id="enable_payment_method" name="enable_payment_method" onchange="valueChanged()">
+    </div>
+
+    <div id="payment_method_fields" style="display: none; margin-bottom: 20px;
+    padding: 20px; border: 1px solid #ccc; border-radius: 4px">
+        <label for="payment_method_type">Payment method type:</label>
+        <select id="payment_method_type" name="payment_method_type">
+            <option value="CARD">Card</option>
+            <option value="ALIPAY">AliPay</option>
+            <option value="WECHAT">WeChat</option>
+        </select>
+
+        <div style="display: flex; align-items: flex-end; margin-top:10px;">
+            <label for="activate_qr_code_scanner">Activate QR code scanner: </label>
```

Comment:
This is not built yet on terminal, let's update documentation only when it's ready